index.html

```diff
@@ -131,6 +135,25 @@ <h1>Connect with me!</h1>
       </ul>
     </section>
 
+    <section id="form">
+      <h2>Leave a Message</h2>
+      <form name="leave_message">
+        <label for="name">Name:</label><br /><br />
```

Comment:
```diff
        <label for="name">Name:</label><br>
```
This is a small point, but you don't really need the `/` in `<br/>`. HTML5 is not a strict subset of XML. Also, it's good not to overuse br: For the spacing between sections, it's better to use CSS. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br 

But neither of these is a big deal!